annotation/app/tasks/resources.py

```diff
@@ -949,6 +957,91 @@ def finish_task(
                 detail=f"Error: connection error ({exc.exc_info})",
             )
         update_inner_job_status(db, task.job_id, JobStatusEnumSchema.finished)
+
+    elif (
+        job.validation_type == ValidationSchema.extensive_coverage
+        and os.getenv("AGREEMENT_SCORE_ENABLED", "false") == "true"
```

Comment:
May we move it to constant var as you did on line 99 for agreement score?

Reply:
it is a dummy temporary solution to make this block of code unreachable until external `agreement score service` implemented. Once it's done we should just remove second condition `and os.getenv("AGREEMENT_SCORE_ENABLED", "false") == "true"`
I've added a comment to make it clear